metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchEventHandlers.java

```diff
@@ -247,8 +247,8 @@ public void metacatUpdateTablePostEventHandler(final MetacatUpdateTablePostEvent
     private void updateEntitiesWithSameUri(final String metadataType, final TableDto dto,
                                            final MetacatRequestContext metacatRequestContext) {
         if (dto.isDataExternal()) {
-            final List<String> ids = es.getTableIdsByUri(metadataType, dto.getDataUri());
-            ids.remove(dto.getName().toString());
+            final List<String> ids = es.getTableIdsByUri(metadataType, dto.getDataUri())
+                .stream().filter(s -> !s.equals(dto.getName().toString())).collect(Collectors.toList());
```

Comment:
is `getName()` not already a string? Seems like the call to `toString()` is redundant but I'm not sure of the return type.